lib/logging/logging.css

```diff
@@ -13,3 +13,24 @@ td.log-label-column {
     padding-left: 5px;
     padding-right: 5px;
 }
+
+.logging-container {
```

Comment:
We may be able to get by with slightly fewer properties here and for `.logging-panel` (I think we can rely on some of the existing flexbox attributes we've defined). Happy to patch this in locally and try and minimize the explicit width / height usages here.